src/test/java/org/jacp/repository/QuestionRepositoryTest.java

```diff
@@ -33,4 +33,13 @@ void testFindById() {
             assertEquals("Grasshopper", entity.getProblem());
         });
     }
+
+    @Test
+    void testFindByIdNegative() {
+        Optional<QuestionEntity> optionalQuestion = questionRepository.findById(QUESTION_ID);
+        assertTrue(optionalQuestion.isPresent());
```

Comment:
assertFalse?

Reply:
This scenario includes checking for an invalid problem value

Reply:
Got it, it's potentially not required if you already checked correct value of problem, it looks like just test for test. Such scenarious should be avoided when possible.

Reply:
Ok
Fixed